packages/auth-utils/migrations/20191103130800-authtoken-blacklist.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const tableName = 'authtokens_blacklist'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable(tableName, {
+      auth_token: {
+        allowNull: false,
+        type: Sequelize.TEXT,
+        primaryKey: true
+      },
+      revoked_by: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      reason: {
+        type: Sequelize.STRING
+      },
+      created_at: {
+        allowNull: false,
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.fn('now')
+      },
+      updated_at: {
```

Comment:
Isn't this an insert only table? If yes do we need the updated_at column?
Maybe in the associated model file add a comment explaining its purpose.

Reply:
Yeah, this is more like a append-only log. We don't need the updated_at column. Can remove it.